### PR TITLE
Some more updates for Valgrind

### PR DESCRIPTION
**Pull request by paulfloyd**
formatting
valgrind-devel has been bumped up
initial vgdb invoker

### CODE REVIEW

File: 2022q2/valgrind.adoc [hunk 1]
```diff
@@ -32,17 +36,22 @@ pointers in ECX.
...
-
 For FreeBSD 13.1 / 14.0 there are
+
 * syscall wrappers for funlinkat, copy_file_range, swapoff, shm_open2
 * add K_INFO handling to fcntl
 * add handling for new auxv entries
 * added some default suppressions for DRD and Helgrind
 
+There is now an initial version of vgdb invoker support - this allows
+vgdb to use ptrace to force valgrind to valgrind poll for gdb commands.
```
Comment (pauamma) [inline 2022q2/valgrind.adoc:47]: spurious extra "valgrind"?

File: 2022q2/valgrind.adoc [hunk 2]
```diff
@@ -32,17 +36,22 @@ pointers in ECX.
...
+
 That does not leave much in the way of outstanding issues. I expect that
 14.0 and newer versions of llvm will keep on requiring support. Apart
 from that there is
 
-* vgdb invoker support
+* some small problems with error messages getting the correct source
+information
 * better core dumps (low priority)
 * TLS handling for Helgrind (difficult if not impossible)
```
Comment (pauamma) [inline 2022q2/valgrind.adoc:57]: Missed that the first time, but perhaps expand TLS (thread-local storage, right?) so people for whom it means Transport Layer Security aren't brought up short?

File: 2022q2/valgrind.adoc (hunk 1 above)
Comment (paulfloyd) [reply 2022q2/valgrind.adoc:47]: Yes, the second should be removed.

File: 2022q2/valgrind.adoc (hunk 2 above)
Comment (paulfloyd) [reply 2022q2/valgrind.adoc:57]: Agreed again.

**Review by pauamma — APPROVED** [review]
Contrary to my expectation, ["thread-local storage" isn't clearly dominant ("thread local storage" dominates slightly overall instead at least until 2019)](https://books.google.com/ngrams/graph?content=thread+local+storage%2Cthread+local+storage&year_start=1800&year_end=2019&corpus=26&smoothing=3&direct_url=t1%3B%2Cthread%20local%20storage%3B%2Cc0%3B.t1%3B%2Cthread%20-%20local%20storage%3B%2Cc0#t1%3B%2Cthread%20local%20storage%3B%2Cc0%3B.t1%3B%2Cthread%20-%20local%20storage%3B%2Cc0l).

**Pull request merged**